tests/conftest.py

```diff
@@ -202,3 +202,11 @@ def persisted_task(request):
 def test_assets_directory() -> pathlib.Path:
     """Gets path to test_assets directory."""
     return pathlib.Path(__file__).parent / "test_assets"
+
+
+@pytest.fixture(scope="module")
+def ai_voltage_chan_task(any_x_series_device):
```

Comment:
All tasks have channels, so `ai_voltage_task` is just as descriptive.

Not all AI voltage tasks are on X Series devices. Consider moving your fixture from conftest.py to test_trigger_properties.py so that it can make assumptions about the device model.